libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/RouteAlert.kt

```diff
@@ -5,15 +5,21 @@ import com.mapbox.geojson.Point
 /**
  * Abstract class that serves as a base for all route alerts.
  *
- * @param type type constant describing the alert, especially useful for Java integration.
- * Kotlin integration can be based on inheritance types.
+ * Available alert types are:
+ * - [TunnelEntranceAlert]
+ * - [BorderCrossingAlert]
+ * - [TollCollectionAlert]
+ * - [RestrictedAreaAlert]
+ * - [RestrictedAreaAlert]
+ *
+ * @param type constant describing the alert type, see [RouteAlertType].
  * @param metadata type-safe metadata of each of the event.
  * @param coordinate location of the alert or its start point in case it has a length
  * @param distance distance to this alert since the start of the route
  * @param alertGeometry optional geometry details of the alert if it has a length
  */
-sealed class RouteAlert<Metadata>(
-    val type: RouteAlertType,
+abstract class RouteAlert<Metadata>(
```

Comment:
Are we moving away from using `sealed` to prevent potential future SemVer changes?

Reply:
Yes :(